Packs/CommonScripts/Scripts/TextFromHTML/TextFromHTML.py

```diff
@@ -5,26 +5,57 @@
 import re
 
 
-def get_plain_text(html_regex):
+def get_plain_text(html: str):
     data = ''
-    if html_regex and html_regex.group(0):
-        data = re.sub(r'<.*?>', '', html_regex.group(0))
+    if html:
+        data = re.sub(r'<br>', '\n', html, flags=re.M + re.S + re.I + re.U)
+        data = re.sub(r'<.*?>', '', html, flags=re.M + re.S + re.I + re.U)
```

Comment:
All these RegEx flags seem unneeded here.

Also, I know this isn't your regex but a previous one, but the `?` there is redundant, since `*` also captures cases where the pattern appears 0 times.

```suggestion
        data = re.sub(r'<.*>', '', html)
```

Reply:
I've reduced the flags, but i kept the re.I flag, as i've seen cases, where HTML contained Upper case Tags, and the regex without the flag just wouldn't replace them.
If necessary, this can be stated in the "breaking changes" section, though i'd rather consider this a fix, as for me this was the inteded behavious from the beginning

Reply:
In Regards to the `?` in the tag regex, it is actually necessary here, because in this case, it tells the `*` to be "lazy"  (see https://www.regular-expressions.info/repeat.html).
If we remove it, it actually breaks the script.

Reply:
I see now what you're saying about the "lazy" issue there.
As for the case-insensitive flag, though, I still don't see why it's needed.
`.` captures all ASCII characters, both upper and lower case.

It might be relevant for the `<br />` RegEx if you want to capture `<BR />` for example too, but shouldn't add anything here.

Reply:
You're right, the `.` captures them all.
Not sure, what it was i rember being an issue. I'm reverting that flag there.